.github/workflows/release.yml

```diff
@@ -19,34 +19,26 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-        with:
-          submodules: recursive
 
       - name: 'Setup'
         uses: ./.github/actions/setup
 
       - name: Build
         shell: bash
-        run: yarn build
+        run: pnpm --filter @coinbase/onchainkit build
 
       - name: Set deployment token
         run: npm config set '//registry.npmjs.org/:_authToken' "${{ secrets.NPM_TOKEN }}"
 
-      - name: Get Version
-        id: get_version
-        run: |
-          VERSION=$(grep -o "export const version = '[^']*'" src/version.ts | cut -d"'" -f2)
-          echo "version=$VERSION" >> $GITHUB_OUTPUT
-
       # https://github.com/changesets/action#with-publishing
       - name: Handle Release Pull Request or Publish to npm
         id: changesets
         uses: changesets/action@v1
         with:
-          title: 'chore: version packages v${{ steps.get_version.outputs.version }} 🌊'
-          commit: 'chore: version packages v${{ steps.get_version.outputs.version }} 🌊'
-          publish: yarn release:publish
-          version: yarn release:version
+          title: 'chore: version packages 🔖'
```

Comment:
Make sure this is also using the version number. Should be fixed by getting this branch up to date with main, but verify.